src/main/drivers/pinio.c

```diff
@@ -58,6 +58,8 @@ void pinioInit(const pinioConfig_t *pinioConfig)
             }
             IOConfigGPIO(io, IOCFG_OUT_PP);
             break;
+        default:
```

Comment:
As above, but compiler is unable to generate warning. But only PINIO_CONFIG_MODE_OUT_PP is expected, `default` will confuse reviewers

Reply:
Deleted